lib/internal/Magento/Framework/App/Response/HttpInterface.php

```diff
@@ -16,4 +16,49 @@
      * @return void
      */
     public function setHttpResponseCode($code);
+
```

Comment:
This is backward incompatible from point of view of extension developers. In case they had custom implementation of this interface and used it for their own logic (not passing it to Magento Core methods), they may be broken now.
@antonkril please approve/reject this change.